### PR TITLE
scope the aliases under eshell so they are not defined globally

### PR DESCRIPTION
**Pull request by bling**

### CODE REVIEW

File: layers/shell/packages.el [hunk 1]
```diff
@@ -140,8 +140,8 @@ is achieved by adding the relevant text properties."
       (require 'esh-opt)
 
       ;; quick commands
-      (defalias 'e 'find-file-other-window)
-      (defalias 'd 'dired)
+      (defalias 'eshell/e 'find-file-other-window)
+      (defalias 'eshell/d 'dired)
```
Comment (syl20bnr) [inline layers/shell/packages.el]: Well, this alias is now longer to type than `dired`:-)

**Comment by syl20bnr** [comment]
I did not realise that the alias were globals which sucks.
On the other hand if we add `eshell/

**Comment by bling** [comment]
not sure what you mean?  even though the function name is `eshell/d` you can still type just `d` in eshell.

**Comment by TheBB** [comment]
Thank you! Cherry-picked in develop. You can safely delete your branch.

@syl20bnr eshell looks up aliases under the `eshell/` prefix, so this is harmless.

**Pull request closed without merging**